caddyhttp/browse/browse.go

```diff
@@ -212,6 +219,10 @@ func directoryListing(files []os.FileInfo, canGoUp bool, urlPath string) (Listin
 
 		url := url.URL{Path: "./" + name} // prepend with "./" to fix paths with ':' in the name
 
+		if _, exists := hidden[f.Name()]; exists {
```

Comment:
Does this take into account different representations of the file paths?

Maybe we should export and use https://github.com/mholt/caddy/blob/8653b70c32f9a79f165d9417f4c6c45eebbc6dd1/caddyhttp/staticfiles/fileserver.go#L136 instead.